meson.build

```diff
@@ -7,8 +7,9 @@ project(
                 'cpp_std=gnu++11',
                 'prefix=/usr',
                 'warning_level=2',
+                'buildtype=debugoptimized',
         ],
-        meson_version : '>= 0.57',
+        meson_version : '>= 0.53.2',
```

Comment:
we probably need this as 0.55, but as a separate commit

Reply:
done